pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -276,14 +275,14 @@ func (exp *flowExporter) sendDataRecord(dataRec ipfix.IPFIXRecord, record flowex
 				deltaBytes = int(record.Conn.OriginalBytes) - int(record.PrevBytes)
 			}
 			if deltaBytes < 0 {
-				klog.Warningf("Delta bytes is not expected to be negative: %d", deltaBytes)
+				klog.Warningf("Delta bytes should not be negative: %d", deltaBytes)
```

Comment:
same comment as above: "Byte delta count for connection should not be negative"

Reply:
Done. Thanks!